// [[Reason: Semantically invalid. TSC does not emit .d.ts]] ////

//// [tests/cases/compiler/fakeInfinity3.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,5 +1,14 @@
 
+
+//// [fakeInfinity3.d.ts]
+export declare enum Foo {
+    A = Infinity,
+    B = -Infinity
+}
+export declare const m: Infinity;
+export declare const Infinity = "oops";
+//# sourceMappingURL=fakeInfinity3.d.ts.map
 /// [Errors] ////
 
 fakeInfinity3.ts(15,17): error TS2749: 'Infinity' refers to a value, but is being used as a type here. Did you mean 'typeof Infinity'?
 fakeInfinity3.ts(15,17): error TS4025: Exported variable 'm' has or is using private name 'Infinity'.
